### PR TITLE
fix(images): keep image configurations if build fails (closes #316)

### LINKED ISSUE #316: [bug]: If an image build fails you lose the edits made to the image configuration
Just tried to create a build image using the latest docker build of abstruse. Sadly it failed as I'd made a syntax error in my scripting but the kicker was I had no way to return to the image editing page to correct my mistake.

I had to redo the docker instructions from scratch, which I did in another editor so I could copy and paste should it have gone wrong again.

### PR DESCRIPTION
**Pull request by Izak88**

### CODE REVIEW

File: src/app/components/app-images/app-images.component.ts [hunk 1]
```diff
@@ -136,6 +138,9 @@ export class AppImagesComponent implements OnInit, OnDestroy {
           }
         } else if (output && output.errorDetail) {
           this.zone.run(() => this.terminalInput = `<span style="color:rgb(255,85,85);">${output.errorDetail.message}</span>`);
```
Comment (jkuri) [inline src/app/components/app-images/app-images.component.ts]: I know this is from previous commits but can you unwrap this message from `<span>` please?

Comment (Izak88) [reply src/app/components/app-images/app-images.component.ts]: sure, but then the error description in terminal will be white instead red, is that ok?

Comment (jkuri) [reply src/app/components/app-images/app-images.component.ts]: I think I encountered this a couple of times and it was not parsed as span but as the whole text as it is here. I think its better to be white than with html text inside terminal output.

Comment (Izak88) [reply src/app/components/app-images/app-images.component.ts]: Yes you're probably right, I'll remove span...

Comment (Izak88) [reply src/app/components/app-images/app-images.component.ts]: done

Comment (jkuri) [reply src/app/components/app-images/app-images.component.ts]: thanks.

**Pull request merged**